src/validators/endpoints.py

```diff
@@ -0,0 +1,122 @@
+import asyncio
+import json
+from typing import Any
+
+from pydantic import TypeAdapter, ValidationError
+from starlette.requests import Request
+from starlette.responses import JSONResponse, Response
+from starlette.status import HTTP_400_BAD_REQUEST
+from sw_utils import is_valid_exit_signature
+from web3 import Web3
+
+from src.config.settings import settings
+from src.validators.api.schemas import ValidatorApproval
+from src.validators.database import NetworkValidatorCrud
+from src.validators.execution import get_latest_network_validator_public_keys
+from src.validators.tasks import (
+    get_available_validators_for_registration,
+    pending_validator_registrations,
+    register_validators,
+)
+from src.validators.typings import Validator, ValidatorsRegistrationMode
+
+
+async def get_validators(request: Request) -> Response:
+    deposit_data = request.app.state.deposit_data
+
+    validators = await get_available_validators_for_registration(
```

Comment:
`check_deposit_data_root` with contact call called in `get_available_validators_for_registration`
MB refactor get_available_validators_for_registration to speed up api response.